collection_processing.py

```diff
@@ -73,9 +76,17 @@ def main():
     stop_words = load_stop_words(stop_words_path)
 
     collection = get_collection_from_corpus(corpus, stop_words)
-    print(collection)
+
+    pickle_save_data_to_file(
+        collection, config.get('collection_path', 'collection_path'))
+    collection_loaded = pickle_load_from_file(
+        config.get('collection_path', 'collection_path'))
+
+    for term in collection:
+        if term not in collection_loaded or collection[term] != collection_loaded[term]:
+            raise Exception(
+                "There is a discrepancy between the collection and the loaded data")
```

Comment:
Is this just to check the save and load are done correctly ?

Reply:
Yes